models/game.rb

```diff
@@ -55,10 +55,23 @@ class Game < Base
     ) personal_games
   SQL
 
+  INVITE_QUERY = <<~SQL
```

Comment:
i don't like how we do a full query here

maybe instead, you should make an entirely new page dedicated to viewing unstarted games like in rolling stock.net

Reply:
Well, that was my initial idea, but I was having trouble figuring out how to grab the game data in that page, so I fell back to this one.

Meanwhile I have learned a bit more about how everything is rendered, so I'll try that way again

Reply:
In any case, the query is looking for an id=-1, which should abort pretty fast

Reply:
I tried doing another page, but it's complicated. If I replace the `Game.home_games` buy something else, it just gets rewritten because there is a `user_refresh` on `assets/app/app.rb`. I tried completely replacing the `assets/app/app.rb` and not use `GameManager`, but then I lose all the auto updating logic, so we'd need to rewrite at least `GameCard` and some other stuff to handle this in a different way.

I don't know what's a better solution.

Reply:
you can get game data easily from the games api we can chat on slack about details